build.assets/Makefile

```diff
@@ -238,7 +238,7 @@ buildbox-arm-fips: buildbox-fips
 grpc: devbox
 	docker run \
 		$(DOCKERFLAGS) -e CLANG_FORMAT=/usr/bin/clang-format-10 -t $(DEVBOX) \
-		make -C /go/src/github.com/gravitational/teleport DISABLE_TEST_TARGETS=1 devbox-grpc
+		make -C /go/src/github.com/gravitational/teleport devbox-grpc
```

Comment:
This change looks unrelated

Reply:
This reverts the change I made in #9432 because I thought that the performance problems were caused by some quirks in the docker vfs bindmounts, so I was only excluding the test targets for the inner part of `make grpc`; with the change in the makefile, this isn't necessary anymore.

I'll note this in the PR description.